Modules/Scripted/VPAWModel/Resources/UI/VPAWModel.ui

```diff
@@ -11,6 +11,32 @@
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QPushButton" name="HomeButton">
+     <property name="enabled">
+      <bool>true</bool>
+     </property>
+     <property name="toolTip">
+      <string>Home</string>
+     </property>
+     <property name="text">
+      <string>Home</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="VPAWVisualizeButton">
```

Comment:
Instead of the text being merely "Home", what if we change it to "Go to Home module", and similarly for the other module buttons?  That would make them harder to confuse as titles for the modules.

I could also move these buttons to below the input and output widgets, though I suppose the user might then lose track of them when the window isn't very tall -- scrolling might be required in that case.

Reply:
I made some changes and pushed them... LMK.

Reply:
I just checked it out -- the buttons look a lot less confusing now! I think this solution works.